src/software/ai/hl/stp/play/ball_placement_play.cpp

```diff
@@ -15,7 +16,9 @@ BallPlacementPlay::BallPlacementPlay(const TbotsProto::AiConfig &config,
 void BallPlacementPlay::getNextTactics(TacticCoroutine::push_type &yield,
                                        const World &world)
 {
-    auto place_ball_tactic = std::make_shared<DribbleTactic>(ai_config);
+    auto place_ball_tactic = std::make_shared<TypedAssignedSkillTactic<DribbleSkillFSM>>(
```

Comment:
we should have a macro that automatically creates tactic versions of skills. A version of this: https://github.com/UBC-Thunderbots/Software/blob/5c40f21b067e8f1aaa2794005d43c1aa8bd8444d/src/software/ai/hl/stp/tactic/tactic.h#L15

Reply:
Made AssignedSkillTactic easier to use, so hopefully we don't need a macro

https://github.com/UBC-Thunderbots/Software/blob/592041e3279f16f759aae0242d8f48dff865ffd5/src/software/ai/hl/stp/play/ball_placement_play.cpp#L20